azure-functions/acbs-function/acbs-amend-facility-master-record/index.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Facility Master Record amendment SOF
+ * **********************************
+ * This is a sub-orchestrated function invoked from it's master orchestrator.
+ * `acbs-amend-facility` is the master orchestrator for this function.
+ *
+ * Sub-Orchestration Function (SOF)
+ * --------------------------------
+ * This function cannot be invoked directly and must be invoked by a DOF.
+ *
+ * Prerequisites
+ * -------------
+ * 0. 'npm install durable-functions'
```

Comment:
Do we need this line?

Reply:
Yep, especially for a clear documentation.